### PR TITLE
Add rules overrides for Wazuh 2.0

### PR DESCRIPTION
**Pull request by max-lobur**
I included an override which shuts down the two rules which spammed - as an example.
links:
https://documentation.wazuh.com/current/user-manual/ruleset/custom.html#changing-an-existing-rule
http://ddpbsd.blogspot.com/2010/10/ossec-rules-101.html

Fixes verygood-ops/infra-vault#511

### CODE REVIEW

**Comment by max-lobur** [comment]
Also, I wasn't able to reproduce that wall of alerts to check if my change works. Anyone recall how to trigger that?

File: files/local_rules.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!-- Ansible managed! -->
+<!-- Overrides built-in rules (git@github.com:wazuh/wazuh-ruleset.git) -->
+<!-- https://documentation.wazuh.com/current/user-manual/ruleset/custom.html#changing-an-existing-rule -->
+
+<group name="ossec,pam,">
+  <rule id="510" level="7" overwrite="yes" noalert="1">
+    <if_sid>509</if_sid>
```
Comment (mjallday) [inline files/local_rules.xml]: this is for the following message

```
OSSEC HIDS Notification.
2017 Jun 03 03:05:12

Received From: (ip-10-5-32-65) any->rootcheck
Rule: 510 fired (level 7) -> "Host-based anomaly detection event (rootcheck)."
Portion of the log(s):

File '/dev/.blkid.tab' present on /dev. Possible hidden file.



 --END OF NOTIFICATION
```

is it possible to do this only for this particular file? it's possible that this message could be useful for other files.

**Comment by max-lobur** [comment]
I added a regex which should filter out blkid files, but didn't test yet. Need to figure out how to do this in an easy way - wazuh docs should help

**Comment by akhon** [comment]
@max-lobur could you please change name of PR to make sense that these changes related to ossec-wazuh 2.0?

**Comment by max-lobur** [comment]
done

**Comment by max-lobur** [comment]
Currently muting alerts via rollbar

**Pull request closed without merging**